AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_tools_utils.py

```diff
@@ -49,6 +53,29 @@ def compare_colors(color1: azlmbr.math.Color, color2: azlmbr.math.Color, buffer:
     )
 
 
+def verify_one_material_document_opened(
+        material_document_ids_list: [azlmbr.math.Uuid], opened_document_index: int) -> bool:
+    """
+    Validation helper to verify if the document at opened_document_index value in the material_document_ids list
+    is the only opened document.
+    Returns True on success, False on failure.
+
+    :param material_document_ids_list: List of material document IDs used for making the document opened check.
+    :param opened_document_index: Index number of the one material document that should be open
+    :return: bool
+    """
+    if not is_document_open(material_document_ids_list[opened_document_index]):
+        return is_document_open(material_document_ids_list[opened_document_index])
```

Comment:
this is confusing why not return a static bool True or False as you mean for this outcome.
if the expected open document is not open (not False) then return document open status (False) why not just return False as a static here so no one has to do the boolean logic in their head to read this code.